homeassistant/components/goalzero/__init__.py

```diff
@@ -82,6 +90,24 @@ def __init__(self, api, coordinator, name, server_unique_id):
         self._server_unique_id = server_unique_id
         self._device_class = None
 
+    @property
+    def extra_state_attributes(self):
+        """Return the state attributes."""
+        attributes = {
+            ATTR_ATTRIBUTION: ATTRIBUTION,
+            "wifi_ssid": self.api.data.get("ssid"),
+            "ip_address": self.api.data.get("ipAddr"),
+        }
+        if (
+            ATTR_FOREIGN_ACCESSORY in self.api.data
+            and self.api.data[ATTR_FOREIGN_ACCESSORY] is not None
+        ):
+            attributes["accessory"] = self.api.data[ATTR_FOREIGN_ACCESSORY][ATTR_MODEL]
+            attributes["accessory_firmware"] = self.api.data[ATTR_FOREIGN_ACCESSORY][
```

Comment:
Firmware version should be part of device info and not state attributes.

Reply:
This is information about an accessory that the user can change at will. We already have model and firmware for the main device.